frontend/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# syntax = docker/dockerfile:1
+
+# Adjust NODE_VERSION as desired
+ARG NODE_VERSION=20.11.1
+FROM node:${NODE_VERSION}-slim as base
+
+LABEL fly_launch_runtime="Next.js"
+
+# Next.js app lives here
+WORKDIR /app
+
+# Set production environment
+ENV NODE_ENV="production"
+
+
+# Throw-away build stage to reduce size of final image
+FROM base as build
+
+ENV NEXT_TELEMETRY_DISABLED 1
+ARG NEXT_PUBLIC_GRAPHQL_ENDPOINT
+ARG NEXT_PUBLIC_CHAIN_ID
+
+# Install packages needed to build node modules
+RUN apt-get update -qq && \
+    apt-get install --no-install-recommends -y build-essential node-gyp pkg-config python-is-python3
```

Comment:
The argument `python-is-python3` called my attention. Is it really needed? 🤔

Reply:
I used a Dockerfile generator suggested on the fly.io documentation.

https://fly.io/docs/js/frameworks/nextjs/#generating-your-nextjs-dockerfile